.env.example

```
@@ -77,6 +77,7 @@ KPIS_NOTION_DATABASE_ID=KPIS_NOTION_DATABASE_ID
 APEX_API_BASE_URI=APEX_API_BASE_URI
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APEX_API_BASE_URI key should go before the API_SECURITY_SCORECARDS_URL key

(UnorderedKey)